skimage/data/_binary_blobs.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+from ..filters import gaussian_filter
+
+
+def binary_blobs(length=512, blob_size_fraction=0.1, n_dim=2,
+                 volume_fraction=0.5, seed=None):
+    """
+    Generate synthetic binary image with several blob-like rounded objects.
+
+    Parameters
+    ----------
+    length : int, default 512
```

Comment:
afaic we say `int, optional`, and leave the default to introspection? Otherwise it increases the chances that the doc and the function diverge.